osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/misc/FindEventsCommandExecutor.java

```diff
@@ -65,7 +65,6 @@
         EVENT_LOG_CATEGORY_OBISCODE_MAP.put(EventLogCategoryDto.COMMUNICATION_SESSION_LOG, new ObisCode("0.0.99.98.4.255"));
         EVENT_LOG_CATEGORY_OBISCODE_MAP.put(EventLogCategoryDto.M_BUS_EVENT_LOG,           new ObisCode("0.0.99.98.3.255"));
         EVENT_LOG_CATEGORY_OBISCODE_MAP.put(EventLogCategoryDto.POWER_QUALITY_EVENT_LOG,   new ObisCode("0.0.99.98.5.255"));
-        EVENT_LOG_CATEGORY_OBISCODE_MAP.put(EventLogCategoryDto.POWER_FAILURE_EVENT_LOG,   new ObisCode("1.0.99.97.0.255"));
```

Comment:
Shouldn't it be possible to get this type of events with the FindEvents request?